app/src/main/java/com/example/easysurvey/ui/theme/RetrofitClientInstance.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.easysurvey.ui.theme
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitClientInstance {
+    private var retrofit: Retrofit? = null
+    //add the url here
+    private val BASE_URL = ""
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere? 
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.